pkg/datapath/loader/base.go

```diff
@@ -262,7 +263,22 @@ func (l *Loader) reinitializeXDPLocked(ctx context.Context, extraCArgs []string)
 	if option.Config.XDPMode == option.XDPModeDisabled {
 		return nil
 	}
+
 	for _, dev := range option.Config.GetDevices() {
+
+		// Don't enable XDP on links that match the option.Config.XDPIgnoreDeviceNameRegex regex
+		// This is useful for when you want XDP but a required device driver on the system does
+		// not yet have support for XDP in the kernel.
+		if len(option.Config.XDPIgnoreDeviceNameRegex) > 0 {
+
+			ignored_interface_re, _ := regexp.Compile(option.Config.XDPIgnoreDeviceNameRegex)
```

Comment:
Nit to have Golang style variables:
```suggestion
			ignored, _ := regexp.Compile(option.Config.XDPIgnoreDeviceNameRegex)
```
Also could you remove some of the blank lines in this `if` block?